src/OrchardCore.Modules/OrchardCore.Liquid/Filters/LocalTimeZoneFilter.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace OrchardCore.Liquid.Filters
 {
-    public class TimeZoneFilter : ILiquidFilter
+    public class LocalTimeZoneFilter : ILiquidFilter
```

Comment:
Breaking change!!

Reply:
How can it be? We register the "local" filter in the startup.cs so renaming the class is not a big deal. And it is registered for every tenants. Why would someone use this class and derivate it?

Reply:
It's a module, at least we need to document this as we already did in some other modules